install/04_deployment.yaml

```diff
@@ -71,9 +71,9 @@ spec:
           secretName: cluster-autoscaler-operator-cert
           items:
           - key: tls.crt
-            path: cert.pem
+            path: tls.crt
           - key: tls.key
-            path: key.pem
```

Comment:
does this belong to this commit?

Reply:
Yes, these file names are actually hard coded in controller-runtime and they changed to match the keys in Kubernetes TLS secrets recently.